src/components/Workspace.jsx

```diff
@@ -95,7 +112,10 @@ class Workspace extends React.Component {
       '_handleRequestedLineFocused',
       '_handleNotificationDismissed',
       '_handleExportGist',
+      '_storeDividerRef',
+      '_storeColumnRef',
     );
+    this.columnRefs = new Map([[1, null], [2, null], [3, null]]);
```

Comment:
Also, I think I had proposed in the previous PR that we store refs in a `Map`—however I think I was thinking about having that map be keyed by the language (of the editors) rather than by index. At this point I definitely agree that it makes more sense to key by index, but with that in mind it’s not clear that a `Map` is better than just using an `Array`…